clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/BootstrappingVeniceChangelogConsumerDaVinciRecordTransformerImpl.java

```diff
@@ -115,24 +121,23 @@ public BootstrappingVeniceChangelogConsumerDaVinciRecordTransformerImpl(Changelo
 
   @Override
   public CompletableFuture<Void> start(Set<Integer> partitions) {
```

Comment:
I think we should make start() synchronized or have some concurrency control to make it thread safe.